packages/e2e/features/ui/components/liveness/disable-start-screen.feature

```diff
@@ -5,11 +5,11 @@ Feature: Disable Start Screen
   Background:
     Given I'm running the example "ui/components/liveness/disable-start-screen/"
   
-  @react
+  @react @liveness-react
```

Comment:
Should we remove `@react` here and on line 12?

Reply:
I updated the tag to default to be @react and not @liveness-react but you may be right. We will never want to run the tests together at least until we MV upgrade liveness